compiler/souschef/include/souschef/Data/Gaussian.h

```diff
@@ -121,6 +121,22 @@ class GaussianInt8DataChef final : public DataChef
   float _stddev;
 };
 
+class GaussianUint4DataChef final : public DataChef
+{
+public:
+  GaussianUint4DataChef(float mean, float stddev) : _mean{mean}, _stddev{stddev}
+  {
+    // DO NOTHING
+  }
+
+public:
+  std::vector<uint8_t> generate(int32_t count) const override;
+
+private:
+  float _mean;
+  float _stddev;
```

Comment:
```suggestion
  float _mean = 0.0;
  float _stddev = 0.0;
```
to make SE tools happy